rbc/tests/test_external.py

```diff
@@ -0,0 +1,224 @@
+import pytest
+from rbc.tests import omnisci_fixture
+from rbc.external import external
+from numba import types
+import numpy as np
+import math
+
+
+@pytest.fixture(scope="module")
+def omnisci():
+
+    for o in omnisci_fixture(globals()):
+        define(o)
+        yield o
+
+
+cmath = (
+    # Trigonometric
+    ("cos", "float64(float64)"),
```

Comment:
Someday, I'd like to move these `(name, signature)` pairs to libfuncs.py.  It is not relevant for this PR though.